tests/bluetooth/bsim_bt/bsim_test_l2cap/prj.conf

```diff
@@ -0,0 +1,33 @@
+CONFIG_BT=y
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_SMP=y
+CONFIG_BT_BONDABLE=y
+CONFIG_BT_ATT_PREPARE_COUNT=8
+CONFIG_BT_GATT_CLIENT=y
+CONFIG_BT_GATT_DYNAMIC_DB=y
+CONFIG_BT_L2CAP_DYNAMIC_CHANNEL=y
+CONFIG_BT_DEVICE_NAME="ECFC test"
+CONFIG_BT_EATT=n
+CONFIG_BT_L2CAP_ECRED=y
+
+# TODO Figure out correct settings
+CONFIG_BT_L2CAP_TX_MTU=2000
+CONFIG_BT_BUF_ACL_RX_SIZE=69
+CONFIG_BT_L2CAP_TX_BUF_COUNT=4
+CONFIG_BT_BUF_ACL_TX_COUNT=4
+CONFIG_BT_BUF_ACL_TX_SIZE=69
+CONFIG_BT_CTLR_RX_BUFFERS=2
+CONFIG_BT_CTLR_DATA_LENGTH_MAX=69
+
+CONFIG_LOG=y
+
+CONFIG_BT_DEBUG_LOG=n
+CONFIG_BT_DEBUG_CONN=n
+CONFIG_BT_DEBUG_L2CAP=n
+CONFIG_BT_DEBUG_KEYS=n
+CONFIG_BT_DEBUG_HCI_CORE=n
+CONFIG_NET_BUF_LOG=n
+CONFIG_NET_BUF_LOG_LEVEL_DBG=n
```

Comment:
These are all `n` by default, and doesn't need to be disabled explicitly. 

In my experience with these tests it is, however, a good idea to keep debug logs enabled for the features that you test, e.g. I'd enable `CONFIG_BT_DEBUG_L2CAP`.